handler/rfc8628/strategy.go

```diff
@@ -0,0 +1,32 @@
+// Copyright © 2023 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+package rfc8628
+
+import (
+	"context"
+
+	"github.com/ory/fosite"
+)
+
+type RFC8628CodeStrategy interface {
```

Comment:
all exported interfaces need godocs in this file.. also.. could we have a single `interfaces.go` file that would contain all the interface definitions? i don't see much point in splitting them up

Reply:
Same as above, your comment makes sense (and it is probably the correct approach). BUT I think if we want to merge this with upstream we should follow the project's practices and all the other handlers are structured this way, so if that's ok I would like to keep this as it is.